website/client/components/settings/subscription.vue

```diff
@@ -237,6 +237,10 @@ export default {
         amount: this.numberOfMysticHourglasses,
       };
     },
+    dateTerminated () {
+      if (!this.user.preferences || !this.user.preferences.dateFormat) return moment(this.user.purchased.plan.dateTerminated);
+      return this.user.purchased.plan.dateTerminated.format(this.user.preferences.dateFormat.toUpperCase());
```

Comment:
`this.user.purchased.plan.dateTerminated` shouldn't be `moemnt(this.user.purchased.plan.dateTerminated)`?

Reply:
I think there's still an issue, line 242 should be `return moment(this.user.purchased.plan.dateTerminated).format(this.user.preferences.dateFormat.toUpperCase());`, otherwise you'll be calling `.format` on a string, which doesn't work

Reply:
You are right. Only line 241 should return normal string object?

Reply:
yes!